.github/workflows/pack-and-release.yml

```diff
@@ -160,7 +164,7 @@ jobs:
           if [ -z $shortSHA ]; then
             shortSHA=`git rev-parse --short=7 HEAD`
           fi
-          yarn oclif promote --deb --xz --root="./packages/cli"  --version=${{ inputs.version }} --sha=$shortSHA
+          yarn oclif promote --deb --xz --root="./packages/cli" --indexes --version=${{ inputs.version }} --sha=$shortSHA --channel=${{ inputs.channel }}
```

Comment:
Is the use of the `--indexes` flag here because we're not getting the promoted urls in the index files?